src/components/MyPortfolio/MyPortfolio.tsx

```diff
@@ -14,40 +15,47 @@ const MyPortfolio: React.FC = () => {
   const { account } = usePangolinWeb3();
   const chainId = useChainId();
 
-  const [showBalances, setShowBalances] = useState(true);
+  const [showUserBalances, setUserShowBalances] = useShowBalancesManager();
+  const [showBalances, setShowBalances] = useState(showUserBalances);
```

Comment:
why do we need extra `useState` here when boolean is already coming from redux ?